Sources/MySQLNIO/Protocol/MySQLProtocol+HandshakeV10.swift

```diff
@@ -103,13 +103,13 @@ extension MySQLProtocol {
                 guard let reserved1 = packet.payload.readSlice(length: 6) else {
                     throw Error.missingReserved
                 }
-                assert(reserved1.isZeroes, "invalid reserve 1 \(reserved1)")
+                assert(reserved1.readableBytesView.allSatisfy { $0 == 0 }, "invalid reserve 1 \(reserved1)")
```

Comment:
How does this change improve performance?

Reply:
I never said this specific change does so 😛. That's just so I could ditch the stupid `isZeroes` property and annoy you by taking your question too seriously 😁

Reply:
@calebkleveter `assert` performance isn't exactly high priority either